tests/conftest.py

```diff
@@ -7485,6 +7487,13 @@ def factory(
                         "dr_workload_app_placement_name"
                     ],
                     discovered_apps_multi_ns=multi_ns,
+                    pvc_interface=pvc_interface,
+                    mix_workload_rbd_pvc_count=workload_details.get(
```

Comment:
I think this will raise exception when the `workload_key` is `dr_workload_discovered_apps_cephfs` or `dr_workload_discovered_apps_rbd`. You can make the get to default to None